flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -730,7 +715,8 @@ private PendingCheckpoint createPendingCheckpoint(
 
         if (statsTracker != null) {
             PendingCheckpointStats callback =
-                    statsTracker.reportPendingCheckpoint(checkpointID, timestamp, props);
+                    statsTracker.reportPendingCheckpoint(
```

Comment:
Should we maybe use `tasksToWaitFor` here instead of `tasksToCommit`? I think in the future the two numbers might differ but `tasksToCommitTo` will strictly have to be a subset of `tasksToWaitFor` because it might be that not all participating tasks need the "notify-complete".

Reply:
Got it, I agree with this. Since the master has been evolved to use the `tasksToWaitFor`, so I just kept the latest behavior in this PR.